### DISCUSSION TITLE
MySTJS and Jupytrerlite/Pyodide

### DISCUSSION THREAD

**Discussion by fpopineau** [executablebooks/meta#1023] [General] [answered]
Hi,

The doc mentions that it is possible to use Jupyterlite to integrate Jupyter notebooks by using:
```
project:
    thebe:
        lite: true
```
in the `myst.yml` configuration file. 
Did anybody managed to do that?
The documentation is scarce and I wonder if it is a WIP or 
if it can work at the moment.

Thanks for your examples or comments.

**Comment by rowanc1** [comment, marked as answer]
@stevejpurves is working on this currently! I am hoping that in the next week and a half it should be working, there is a branch of the work here:
https://github.com/executablebooks/myst-theme/pull/70
Very excited about the possibilities here. 🚀

**Reply by fpopineau** [reply]
Ahah! I did not look at this repo yet :-)
Yes, that would be great!

**Reply by stevejpurves** [reply]
yes - I'm working through some regressions on that PR - hopefully it will be in soon!
